app/models/concerns/ransackable_attributes.rb

```diff
@@ -0,0 +1,13 @@
+module RansackableAttributes
```

Comment:
Is this correct name for concern as it includes both attributes and associations?